packages/tslint-config/react-native.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+
+const path = require('path');
+
+module.exports = {
+  rulesDirectory: [
+    path.join(path.dirname(require.resolve('tslint-microsoft-contrib')), './'),
+    path.join(path.dirname(require.resolve('tslint-origin-ordered-imports-rule')), './'),
+  ],
+  rules: {
+    quotemark: [
+      true,
+      'single',
+      'jsx-double',
+    ],
+
+    'origin-ordered-imports': [
+      true,
+      'one-blank-line',
+    ],
+
+    'max-line-length': [
+      true,
+      {
+        limit: 140,
+        'ignore-pattern': '^import |^export {(.*?)}',
+      },
+    ],
+
+    'no-implicit-dependencies': [
+      true,
+      'dev',
+      [
+        '~assets',
+        '~components',
+        '~config',
+        '~stores',
+        '~utils',
+        '~screens',
```

Comment:
This could be problematic, let's either
a) remove this rule altogether, or
b)keep it with [true, 'dev'], and add the list of aliases in the starter kit for easy modification of the list.

Use case: Someone may want to add his own folder, ex. `~graphql/listUsers.gql`, and this is not going to be an easy find from the project perspective.

Reply:
I totally agree I was thinking about it. It's probably safer to completely remove it from the config and put it on the starter kit instead.